scripts/calico.yaml

```diff
@@ -85,14 +95,21 @@ spec:
       hostNetwork: true
       containers:
         - name: calico-etcd
-          image: gcr.io/google_containers/etcd-amd64:3.1.10
+          image: quay.io/coreos/etcd:v3.1.10
```

Comment:
The reason for the explicit change here was to reduce the image count.  Otherwise, there are 2 images that are essentially the same.

Reply:
just a heads up 1.9.2 uses 3.1.11 https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.9.md#changelog-since-v191

```
$ k describe po -n kube-system etcd-ip-10-0-4-240.us-west-1.compute.internal | grep Image:
    Image:         gcr.io/google_containers/etcd-amd64:3.1.11
```
I couldn't find an exact version calico requires, but it might just be v3.x

That being said, I don't think there's anything wrong with similar but different images on the host. Did we run into any issues with too many images? I wonder if this was a premature optimization or if we got bit by it.

Reply:
I'd still change this to gcr.io/google_containers/etcd-amd64:3.1.10 to reduce the container count, it's the same thing.